actioncable/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [
+    ["env", { "modules": false } ]
```

Comment:
Let's use Babel's loose mode. The resulting JS is smaller and more like what the CoffeeScript version produced.
```diff
--- a/actioncable/.babelrc
+++ b/actioncable/.babelrc
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["env", { "modules": false } ]
+    ["env", { "modules": false, "loose": true } ]
   ],
   "plugins": [
     "external-helpers"
```
Full diff: https://gist.github.com/javan/387bec1ca052e36a9904177f87fed2de

```
app/assets/javascripts/action_cable.js
  before: 19664 bytes
   after: 17900 bytes
```

Reply:
(I plan to do the same for Active Storage too)

Reply:
Done in https://github.com/rmacklin/rails/commit/746a19c9127fbda263571faa2e2f138e93e0cd3d